tox.ini

```diff
@@ -37,7 +38,7 @@ commands=
     integration-parity-ipc: pytest -s {posargs:tests/integration/parity/test_parity_ipc.py}
     integration-parity-http: pytest -s {posargs:tests/integration/parity/test_parity_http.py}
     integration-parity-ws: pytest -s {posargs:tests/integration/parity/test_parity_ws.py}
-    doctest: make -C {toxinidir}/docs doctest
+    doctest: make -C {toxinidir} validate-and-test-docs
```

Comment:
Probably good to rename `doctest` to just `docs` to avoid confusion. The term `doctest` seems loaded with the `doctest` directive to test code examples in the docs. Renaming to `docs` might convey better that this is a job that validates the entire docs from building the html, to running the examples and creating the release notes.